test/gtest/ucp/test_ucp_mmap.cc

```diff
@@ -154,8 +154,6 @@ class test_ucp_mmap : public ucp_test {
                               sender().ucph()->cache_md_map[memh1->mem_type];
         ucp_md_index_t md_index;
 
-        EXPECT_NE(memh1->md_map, 0);
```

Comment:
maybe it is worth to disable some tests which are not relevant anymore for some transports (e.g. cma)

Reply:
IMO it's better to run all transports and don't assume a specific behavior according to a transport name, because it can change